.ci/downstreamTests.groovy

```diff
@@ -160,3 +162,14 @@ def runScript(Map params = [:]){
     }
   }
 }
+
+/**
+* Whether the given ruby version and framework are in charge of sending the
+* codecov results. It does require the workspace.
+*/
+def isCodecovEnabled(ruby, framework){
```

Comment:
The `Enabled` part could be misleading compared to the javadocs, wdyt?

Reply:
Do you mean `isCodecov` rather than `isCodecovEnabled`?